lib/__testfixtures__/object-prototype.input.mjs

```diff
@@ -0,0 +1,23 @@
+// Source from Object prototype
+const elt = document.createElement('div');
+elt.toString();
+elt.valueOf();
+elt.hasOwnProperty('remove');
+
+// Should not source from Object but TypeScript is behind
+// https://github.com/jackbsteinberg/get-originals-rewriter/issues/79
+const url = new URL();
+url.toString();
+const bool = new Boolean();
+bool.toString();
+const te = new TypeError();
+te.toString();
+
+// Don't source from Object prototype
```

Comment:
Can we add tests for strings? In the three variants: `const s = new String()`, `const s2 = ""`, and just `"".valueOf()` directly. (Either valueOf or toString will work.)

Just like numbers, there are some subtleties here around literals vs. the wrapper `String` object type and so on.